src/DataTransfer/infra/excel/downloadExcel.go

```diff
@@ -0,0 +1,62 @@
+package excel
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"os"
+
+	"golang.org/x/net/context"
+
+	"google.golang.org/api/drive/v3"
+	"google.golang.org/api/option"
+)
+
+func downloadExcel(spreadsheetID string, filename string) {
+	ctx := context.Background()
+
+	driveService, err := drive.NewService(ctx, option.WithCredentialsFile("secret.json"))
+	if err != nil {
+		log.Fatal(err, " client")
```

Comment:
lo normal para el formato de error es poner "*causa*: *error*" y por costumbre, como este es un mensaje que termina el programa añade una linea al final con `Fatalln`
```suggestion
		log.Fatalln("client:", err)
```